kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -1098,7 +1108,7 @@ public void onChannelUpdated() {
 
         //If user has Integrated Zopim, initialize Zendesk Chat SDK
         Contact assigneeContact = appContactService.getContactById(channel.getConversationAssignee());
-        if(!TextUtils.isEmpty(zendeskChatSdk) && assigneeContact != null && User.RoleType.AGENT.getValue().equals(assigneeContact.getRoleType()) ) {
+        if(!TextUtils.isEmpty(zendeskChatSdk) && assigneeContact != null && User.RoleType.AGENT.getValue().equals(assigneeContact.getRoleType()) && channel.getKmStatus() != Channel.CLOSED_CONVERSATIONS) {
```

Comment:
is this string comparison? 
.CLOSED_CONVERSATIONS
if yes then use equals

Reply:
That is an  Integer comparison.